common/config.go

```diff
@@ -26,6 +25,21 @@ const (
 var homeDir, _ = os.UserHomeDir()
 var baseDir = path.Join(homeDir, ".atomicswap")
 
+// publicBootnodes are bootnodes with public IP addresses and they used in every
+// environment other than development
+var publicBootnodes = []string{
+	"/ip4/109.105.198.218/tcp/9909/p2p/12D3KooWRbAV4AU75JXVUedmwakiM1yzzDzDCdmZWzGdXGQ4Yqdv",
```

Comment:
There are 2 nodes on port 9909 that didn't have a 9900 entry on the same IP in the former stagenet list. All others that had 2 entries on the same IP were consolidated in favor of the entry on port 9900.